modules/service/src/test/scala/lucuma/odb/graphql/feature/reference.scala

```diff
@@ -147,15 +127,18 @@ class reference extends OdbSuite {
     for {
       pid0 <- createProgramAs(pi)
       _    <- addProposal(pi, pid0)
-      ref0 <- submitProposal(pi, pid0, sem2024B.some)
+      _    <- setSemester(pi, pid0, sem2024B)
```

Comment:
Do we have, or could we add, a test somewhere in which we try to submit a proposal without first having set the semester to make sure it is handled in some reasonable-ish way?

Reply:
I thought I had that, but I guess not. I'll add it.

Reply:
I was wrong. There is one. 4th test in `setProposalStatus`